### PR TITLE
Updated Entity Model Closes #27

### LINKED ISSUE #27: Update the Entity Model to match the ERD.
DOD: When the entity model matches the ERD, is approved by the team, and merged into develop.

### PR DESCRIPTION
**Pull request by weberwilliamb**
- Created Tag entity
- Updated User entity with email
- Updated Data entity with comment
- Added Navigation Properties to each Entity

### CODE REVIEW

File: photo-points/Models/Capture.cs [hunk 1]
```diff
@@ -11,13 +11,10 @@ public class Capture
     public long captureID { get; set; }
     public byte photo { get; set; }
     public DateTime captureDate { get; set; }
-    public long photoPointID { get; set; }
-    public long userID { get; set; }
+    
+    public ICollection<Data> data { get; set; }
```
Comment (Sara-webaplication) [inline photo-points/Models/Capture.cs]: we suppose to have approved,

**Review by Sara-webaplication — APPROVED** [review]

**Review by Sara-webaplication — APPROVED** [review]

**Comment by Sara-webaplication** [comment]
looks great, Thank you

File: photo-points/Models/Capture.cs (hunk 1 above)
Comment (weberwilliamb) [reply photo-points/Models/Capture.cs]: Added in now! Thanks for catching this. Accidentally deleted when I was adding the Navigation properties.

**Comment by RoEric** [comment]
> * Created Tag entity
**> * Updated User entity with email**
> * Updated Data entity with comment
> * Added Navigation Properties to each Entity

I'm not seeing see any email property currently in the User model in your branch.

**Comment by weberwilliamb** [comment]
@RoEric It is updated now.

**Review by RoEric — APPROVED** [review]
Great work.

File: photo-points/Models/Data.cs [hunk 2]
```diff
@@ -8,9 +8,11 @@ namespace photo_points.Models
     public class Data
 {
     public long dataID { get; set; }
-    public long captureID { get; set; }
     public string type { get; set; }
     public string value { get; set; }
+    public string comment { get; set; }
+    
+
     public Capture capture { get; set; }
```
Comment (sruble) [inline photo-points/Models/Data.cs:16]: should this line be
public ICollection<Capture> capture {get; set;}
since its the entity?

**Review by sruble — CHANGES_REQUESTED** [review]
A comment in the Data.cs file. I was wondering if the data type needed to be changed.

File: photo-points/Models/Data.cs (hunk 2 above)
Comment (weberwilliamb) [reply photo-points/Models/Data.cs:16]: No, because data doesn't have a collection of captures.

**Comment by weberwilliamb** [comment]
@sruble: That data type works for the comments field.

**Review by sruble — APPROVED** [review]
looks good

**Review by MaxIgnatyev — APPROVED** [review]
Seems fine to merge.

**Pull request merged**